src/utils.tsx

```diff
@@ -288,17 +288,21 @@ export const sendPerfTiming = ({
   _tracking = tracking,
   actualDuration,
   id,
-  phase,
+  url,
 }: {
   _tracking?: typeof tracking;
   actualDuration: number;
   id: string;
-  phase: string;
+  url: string;
 }) => {
   _tracking.timing({
     category: 'renderPerf',
-    label: phase,
+    label: url,
```

Comment:
Couldn't we set `url` to `location.href` directly here so that it takes the full url from the browser? That would avoid a lot of code.

Reply:
There is no such thing as `location.href`. Is that another thing we have in addons-frontend, but not in code-manager?

Reply:
ah well, `window.location.href` that is

Reply:
Ah, I was just checking what we had available in the `location` object. Yes, we could use `window.location.href`. I'll change that.